save-core/src/commonMain/kotlin/org/cqfn/save/core/utils/ProcessBuilder.kt

```diff
@@ -45,6 +45,11 @@ expect class ProcessBuilderInternal(
  */
 @Suppress("INLINE_CLASS_CAN_BE_USED")
 class ProcessBuilder {
+    /**
+     * Typealias
```

Comment:
That's not a typealias, that's a singleton that describes the current file system

Reply:
Changed